specifyweb/workbench/upload/parsing.py

```diff
@@ -26,7 +26,7 @@ class PicklistAddition(NamedTuple):
 
 class ParseFailure(NamedTuple):
     message: str
-    payload: Dict[str, Any]
+    payload: Dict[str, Union[str, int, List[str], List[int]]]
```

Comment:
Are you sure mypy doesn't support `Any` type? https://mypy.readthedocs.io/en/stable/kinds_of_types.html#the-any-type

What error did you get?

Sorry, I can't get the back-end tests to run on my machine to see the error for myself

Reply:
it's not mypy that is the problem, it's hypothesis that is not able to mock it for testing

Reply:
to run a unit tests locally run ```docker exec -it specify7-specify7-1 ve/bin/python3 manage.py test specifyweb.workbench.upload.tests.test_upload_results_json.UploadResultsTests.testUploadResult```